<!DOCTYPE HTML>
<!--
	Dimension by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Samuel E Miller</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<!-- div class="logo">
							<span class="icon fa-book"></span>
						</div -->
						<div class="content">
							<div class="inner">
								<h1>Samuel E. Miller, PhD </h1>
								<p>Computational and Molecular Microbiology</p>
							</div>
						</div>
						<nav>
							<ul>
								<li><a href="#cv">CV</a></li>
								<li><a href="#research">Research</a></li>
								<li><a href="#teaching">Teaching</a></li>
								<li><a href="#contact">Contact</a></li>
								<!--<li><a href="#elements">Elements</a></li>-->
							</ul>
						</nav>
					</header>

				<!-- Main -->
					<div id="main">

						<!-- CV -->
							<article id="cv">
								<h2 class="major">CV</h2>
								<p>
									Scientific Programmer (2022-)<br />
									<a href="https://ccomp-stc.org/" target="_blank"><u>NSF Center for Chemical Currencies of a Microbial Planet</u></a><br />
									<a href="https://www.mbl.edu/research/research-centers/josephine-bay-paul-center" target="_blank"><u>Bay Paul Center</u></a>, The Marine Biological Laboratory<br />
									<br />
									
									Postdoctoral Scholar (2019-22)<br/>
									Department of Medicine, The University of Chicago<br />
									<a href="http://merenlab.org" target="_blank"><u>A. Murat Eren Lab</u></a><br />
									<br />

									<b>Education</b><br />
									
									<br />
									Ph.D and M.S. (2010-18)<br />
									Department of the Geophysical Sciences, The University of Chicago<br />
									Dissertation: <a href="https://knowledge.uchicago.edu/record/1419" target="_blank"><u>The Metaproteomic Analysis of Arctic Soils with Novel Bioinformatic Methods</u></a><br />
									<br />

									B.A. <i>magna cum laude</i> (2006-10)<br />
									Amherst College, Chemistry and Geology double major (2006-10)<br />
									Thesis: The Microbiology and Geochemistry of the New Albany Shale, Illinois Basin<br />
									(presented almost verbatim in the New Albany Shale Report below under Publications)
								</p>
								<p>
									<b>Publications </b><br />
									* indicates corresponding author<br />
									<br />

									<a href="https://www.biorxiv.org/content/10.1101/2024.07.03.601779v1" target="_blank"><u>Adaptive adjustment of significance thresholds produces large gains in microbial gene annotations and metabolic insights</u></a><br />
									Kananen, K.; Veseli, I.; Quiles Pérez, C.J.; Miller, S.; *Eren, A.M.; *Bradley, P.H.<br />
									bioRxiv preprint, <i>submitted</i> ISME Communications<br />
									<br />

									<a href="https://www.biorxiv.org/content/10.1101/2024.01.16.575828v1" target="_blank"><u>Digital Microbe: A Genome-Informed Data Integration Framework for Collaborative Research on Emerging Model Organisms</u></a><br />
									Veseli, I.; Cooper, Z.S.; DeMers, M.A.; Schechter, M.S.; Miller, S.; Weber, L.; Smith, C.B.; Rodriguez, L.T.; Schroer, W.F.; McIlvin, M.R.; Lopez, P.Z.; Saito, M.; Dyhrman, S.; *Eren, A.M.; *Moran, M.A.; *Braakman, R.<br />
									bioRxiv preprint, <i>in revision</i> Scientific Data<br />
									<br />

									<a href="https://journals.asm.org/doi/full/10.1128/msystems.01238-22" target="_blank"><u>Metaproteomics reveals functional partitioning and vegetational variation among permafrost-affected Arctic soil bacterial communities</u></a><br />
									*Miller, S.E.; Colman, A.S.; *Waldbauer, J.R.<br />
									mSystems (2023), 8 (3)<br />
									<br />

									<a href="https://www.science.org/doi/10.1126/sciadv.abq4632" target="_blank"><u>Structure-informed microbial population genetics elucidate selective pressures that shape protein evolution</u></a><br />
									*Kiefl, E.; Esen, O.C.; Miller, S.E.; Kroll, K.L.; Willis, A.D.; Rappé, M.S.; Pan, T.; *Eren, A.M.<br />
									Science Advances (2023), 9 (8)<br />
									<br />

									<a href="https://academic.oup.com/nar/article/50/17/e99/6609813" target="_blank"><u>Analysis of queuosine and 2-thio tRNA modifications by high throughput sequencing</u></a><br />
									Katanski, C.D.; Watkins, C.P.; Zhang, W.; Reyer, M.; Miller, S.; *Pan, T.<br />
									Nucleic Acids Research (2022), 50 (17), p. e99<br />
									<br />

									<a href="https://www.nature.com/articles/s41564-020-00834-3" target="_blank"><u>Community-led, integrated, reproducible multi-omics with anvi'o</u></a><br />
									*Eren, A.M.; Kiefl, E.; Shaiber, A.; Veseli, I.; Miller, S.E.; Schechter, M.S.; et al.
									Nature Microbiology (2021), 6, pp. 3-6<br />
									<br />
									
									<a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00278"><u>Postnovo: Postprocessing Enables Accurate and FDR-Controlled de Novo Peptide Sequencing</u></a><br />
									*Miller, S.E.; Rizzo, A.I.; *Waldbauer, J.R.<br />
									Journal of Proteome Research (2018), 17 (11), pp. 3671-3680<br />
									<br />

									The New Albany Shale: Formation Water, Gas Geochemistry and the Subsurface Biotic Community<br />
									<i>In</i> New Albany Shale Gas Project Final Report<br />
									*Martini, A.M.; Petsch, S.T.; McIntosh, J.C.; Schlegel, M.; Damashek, J.; Miller, S.E.; Kirk, M.<br />
									Gas Technology Institute, Research Partnership to Secure Energy for America (2010), 07122-16<br />
									<a href="https://www.netl.doe.gov/File%20Library/Research/Oil-Gas/Natural%20Gas/shale%20gas/07122-16-final-report.pdf#page=155"><u>PDF</u></a><br />
									<br />

									<b>Selected Oral and Poster Presentations </b><br />
									<br />

									Integrated high-throughput tRNA sequencing and metagenomics reveal the translational dynamics of the human gut microbiome<br />
									*Miller, S.E.; Katanski, C.; Watkins, C.; Veseli, I.; Fuessel, J.; Gerasimidis, K.; Quince, C.; Raguideau, S.; Pan, T.; Eren, A.M.<br />
									Oral (2022), International Society for Microbial Ecology 18, Lausanne, Switzerland<br />
									<br />

									Novel metaproteomic approaches reveal systematic variations in microbial biogeochemical pathways with Arctic vegetation types<br />
									*Miller, S.E.; Waldbauer, J.R.<br />
									Poster (2018), American Geophysical Union Annual Meeting, Washington, D.C. B31H-2580<br />
									<a href="images/agu_2018_poster.pdf"><u>Poster PDF</u></a><br />
									<br />

									Novel metaproteomic approaches applied to Arctic soils reveal niche partitioning of microbial biogeochemical functions and systematic variations with vegetation type<br />
									*Miller, S.E.; Waldbauer, J.R.<br />
									Oral (2018), Midwest Geobiology Symposium, Northwestern University<br />
									<br />

									Metaproteomic analysis of Arctic soils with Postnovo, a new computational pipeline<br />
									*Miller, S.E.; Rizzo, A.I.; Waldbauer, J.W.<br />
									Oral (2017), Argonne Soil Metagenomics Meeting, Argonne National Laboratory<br />
									<br />

									Proteomics of soil and sediment: protein identification by de novo sequencing of mass spectra complements traditional database searching<br />
									*Miller, S.E.; Rizzo, A.I.; Waldbauer, J.W.<br />
									Oral (2014), American Geophysical Union Annual Meeting, San Francisco, CA. B13K-05.<br />
									<br />

									<b>Awards, Honors, and Additional Experience</b><br />
									<br />

									University of Chicago Division of the Physical Sciences Robert N. Ginsburg Fellowship (2018, 2017, 2014) <br />
									<br />

									Marine Biological Laboratory/University of Chicago Graduate Student Research Award (2015) <br />
									<br />

									University of Chicago Symphony Orchestra Ellis Bohnoff Kohs Award for Orchestral Excellence (2013) <br />
									<br />

									National Science Foundation Graduate Research Fellowship Program Honorable Mention (2012, 2011) <br />
									<br />

									International Geobiology Course (2011) <br />
									<br />

									University of Chicago Division of the Physical Sciences Robert R. McCormick Fellowship (2011, 2010) <br />
									<br />

									Geological Society of America Northeastern/Southeastern Sections Meeting Outstanding Student Presentation Award (2010) <br />
									<br />

									Amherst College Department of Music Sylvia and Irving Lerner Prize (2010) <br />
									<br />

									Amherst College Dean of the Faculty Fellowship for Summer Thesis Research (2009) <br />
									<br />

									University of Houston/Yellowstone-Bighorn Research Association Geology Field Camp (2009) <br />
									<br />

									Amherst College Department of Geology Belt-Brophy Prize (2009) <br />
									<br />

									National Science Foundation Research Experience for Undergraduates Fellowship (2008) <br />
									<br />

									Amherst College Edward Hitchcock Fellowship for Summer Research (2007) <br />
									<br />

									National Merit Scholar (2006-2010) <br />
									<br />
									
									Valedictorian, Apple Valley High School, Apple Valley, MN (2005) <br />
								</p>
							</article>

						<!-- Research -->
							<article id="research">
								<h2 class="major">Research</h2>
								<span class="simple_image main"><img src="images/miller.jpg" alt="" /></span>
								<p>
									I study how microbes live in the wild -- be it in seawater, soils, or inside us -- using multi-omics methods.
									I develop widely applicable bioinformatic software, with particular experience beyond the genome in metaproteomics, tRNA sequencing, and other technologies that reveal what microbes are doing in situ, rather than just what their DNA is capable of doing.
								</p>

								<p>
									Here are a few of my interests.
								</p>

								<h3 class="major">Microbial Activity in Arctic Soils</h3>

								<span class="simple_image main"><img src="images/Toolik_Summary_Figure.jpg" alt="" /></span>

								<p>
									This research was motivated by the question of how greater plant growth associated with Arctic warming affects soil microbial processes critical to the cycling of vast quantities of organic carbon.
									Metaproteomics -- the characterization of proteins expressed by multiple organisms in a sample -- can be used to understand organic matter transformations, intractable to direct chemical measurement, and the taxa responsible for these processes.
									I developed new software tools to enable the application of metaproteomics to complex samples such as soils.
									Results of this research, as summarized below, were published <a href="https://journals.asm.org/doi/full/10.1128/msystems.01238-22" target="_blank"><u>here</u></a>.
								</p>

								<p>
									Metaproteomic analysis of soils that I collected from Toolik Field Station in Arctic Alaska yielded significant insights into the functional differentiation of major bacterial groups and the variation of these taxa and their metabolic activity with vegetation.
									I found that protein expression among &alpha;-/&beta;-/&gamma;-Proteobacteria is structured around the acquisition of small solutes and nitrogen, the limiting nutrient in tundra soils.
									Acidobacteria -- a poorly understood but ubiquitous phylum in soils -- is the most active group by a combination of biomass and per-cell activity, as inferred from the expression of core cellular pathways, and specializes in the degradation of relatively labile polysaccharides such as hemicelluloses and starch.
									Other groups invest in enzymes targeting other polymers -- Actinobacteria depolymerizes cellulose, Burkholderiaceae breaks down lignin, and Bacteroidetes and Myxococcales decompose pectin.
									Each taxonomic group maintains a characteristic protein expression profile between vegetation types.
									&alpha;-/&beta;-/&gamma;-proteobacterial functions, however, are more prevalent in "greener," higher-biomass floras favored in a warmer Arctic.
								</p>

								<p>
									These results indicate a fundamental divide between &alpha;-/&beta;-/&gamma;-proteobacterial taxa adapted to plant interactions in rhizosphere microenvironments and largely non-proteobacterial taxa adapted to polymer degradation in the bulk soil.
									&alpha;-/&beta;-/&gamma;-Proteobacteria dominate the expression of transporters for simple sugars which likely come from root exudation rather than the breakdown of soil organic matter, as these bacteria participate minimally in polysaccharide depolymerization.
									Carbon monoxide is also mainly produced by roots in soils and seems to largely be consumed by &alpha;-/&beta;-/&gamma;-Proteobacteria.
									Although key proteins for intracellular nitrogen cycling are expressed by all of the bacterial groups at levels consistent with overall activity (as inferred by comparison to ribosomal protein expression), &alpha;-/&beta;-/&gamma;-Proteobacteria disproportionately produce transporters for nitrogenous compounds.
									The greater concentration of transporters in rhizospheric &alpha;-/&beta;-/&gamma;-Proteobacteria than bulk soil groups may reflect stiff competition with plants for scarce nitrogen around roots.
									Likewise, rhizospheric bacteria seem to be the major producers of carbon storage polymers (polyhydroxyalkanoates), which are often associated with nutrient starvation.
									Phosphorus transporters are much more evenly expressed across taxa, as expected from a more even concentration of phosphorus across soil microenvironments due to its relative abundance compared to nitrogen.
									Greater microbial nitrogen limitation among rhizospheric than bulk soil taxa supports the <a href=https://iopscience.iop.org/article/10.1088/1748-9326/aaa4fa target="_blank">theory</a> that plant nitrogen uptake and therefore plant growth in the Arctic is limited by microbial competition with plants rather than slow organic matter decomposition.
								</p>

								<p>
									The consistency of microbial protein expression profiles across soils rooted by different plant ecotypes suggests that the functional specialization of bacterial groups does not change with Arctic greening.
									The functional stability of microbial groups is encouraging for the prediction of future biogeochemical trends in a rapidly greening Arctic from experimental manipulations of soils and isolates.
								</p>

								<p>
									Here are results showing the strong metaproteomic signal of functional partitioning by microbial groups.
								</p>

								<b>Cell Growth</b>
								<span class="simple_image main"><img src="images/Toolik_Growth_Heatmap.jpg" alt="" /></span>

								<b>Carbon Metabolism</b>
								<span class="simple_image main"><img src="images/Toolik_Carbon_Heatmap.jpg" alt="" /></span>

								<b>Nutrient Metabolism</b>
								<span class="simple_image main"><img src="images/Toolik_Nutrients_Heatmap.jpg" alt="" /></span>

								<b>Cell Envelope</b>
								<span class="simple_image main"><img src="images/Toolik_Envelope_Heatmap.jpg" alt="" /></span>
								<br \>

								<h3 class="major">Proteomic Software Development</h3>
								<p>
									I have developed Python tools for the analysis of peptide mass spectra and metaproteomic data.
									New computational approaches were essential for interpreting complex soil multi-omic datasets and should prove useful in a variety of environmental and biomedical samples.
									A major challenge that I encountered in my soil analyses was the lack of an appropriate sequence database for identification of peptide mass spectra by standard database search methods.
									For many types of simple proteomic samples, the database consists of protein-coding sequences from the reference genome of the sampled organism.
									The high genomic microdiversity of soil microbes limits the utility of database search even with paired metagenomic datasets, as a single amino acid mutation can significantly alter the mass properties of a peptide.
									The constraint of an appropriate reference database can create challenges in a variety of sample types, not just environmental metaproteomes -- monoclonal antibodies, unexpected sequence variants, nonribosomal and post-translationally modified peptides, and fossil proteins.
								</p>

								<p>
									De novo sequencing is an alternative method to database search that attempts to directly solve the sequence of a peptide mass spectrum without the aid of a reference database.
									I developed a post-processing machine learning tool, Postnovo, to improve the low accuracy of de novo sequences from existing programs, published <a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00278" target="_blank"><u>here</u></a>.
									Postnovo serves the same purpose in de novo sequencing as the widely used tools, <a href="http://percolator.ms/" target="_blank">Percolator</a> and <a href="http://peptideprophet.sourceforge.net/" target="_blank">PeptideProphet</a>, do in database search post-processing.
									Largely through newly computed metrics and the identification of consensus sequences from input de novo sequence candidates by a novel, efficient dynamic programming algorithm, Postnovo increases the yield of accurate de novo sequences at a given FDR by about an order of magnitude.
									Further, Postnovo reliably estimates the posterior error probability of de novo sequences, allowing FDR control.
									See the <a href="https://github.com/semiller10/postnovo" target="_blank">Postnovo GitHub page</a> for details.
								</p>

								<p>
									I developed another tool, ProteinExpress, to better analyze metaproteomic data using database search against metagenomes/transcriptomes, published <a href="https://journals.asm.org/doi/full/10.1128/msystems.01238-22" target="_blank"><u>here</u></a>.
									ProteinExpress is designed to address the problem of reference database incompleteness in peptide-spectrum matching and protein inference.
									A match between a peptide mass spectrum and a protein-coding sequence does not guarantee that the mass spectrum originated from the taxon associated with the protein, as opposed to another taxon producing the same peptide in an unsampled coding sequence.
									ProteinExpress addresses the problem of taxonomic assignment by using a probabilistic metric for protein expression based on sequence similarity to the suite of taxonomic bins of assembled contigs.
									ProteinExpress also increases the peptide identification rate by performing database search against both reads and contigs from multiple metagenomes/transcriptomes.
									In my study of Arctic soils, the identification rate was greatly increased by comparing metaproteomes to 28 publicly available metagenomes/transciptomes collected from different sites in Alaska.
									The lower genomic diversity of colder high-latitude soils than warmer low-latitude soils that has been <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2016.00579/full" target="_blank">observed</a> encouraged me to leverage multiple nucleotide datasets.
									ProteinExpress performs protein functional annotation using <a href="http://eggnogdb.embl.de/#/app/emapper" target="_blank">eggNOG-mapper</a>, and I developed a curated database of broader Functional Groups based on eggNOG terms to facilitate functional interpretation (e.g., Cellulases, Polyamine Synthesis, Glycolysis).
								</p>
								
								<iframe width="560" height="315" src="https://www.youtube.com/embed/F0tVXLPujGI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

								<p><a href="images/dissertation_defense.pdf"><u>Here</u></a> are the slides from my dissertation defense.</p>
							</article>

						<!-- Teaching -->
							<article id="teaching">
								<h2 class="major">Teaching</h2>
								<p>
									I was a teaching assistant in numerous courses at the University of Chicago, including my largest responsibility of designing and teaching labs in Environmental Chemistry.
								</p>

								<p>
									One new lab involves measuring mercury levels in an array of fish by atomic absorption spectroscopy and modeling blood mercury from the data.
									The most interesting part of the lab report combines the measured Hg values, data on fish consumption by the populations of the USA and Norway (the country with the highest fish consumption), and a pharmacokinetic model to ask the students how their blood Hg level would change during a semester abroad in Norway adopting the local diet.
									If and when does their blood Hg cross the threshold that the EPA considers dangerous after moving to Norway and upon returning to the USA?
								</p>
								
								<p>
									The other new lab that I designed involves benchtop microcosms of the <a href="http://www.h2o2.com/industrial/fentons-reagent.aspx?pid=84&name=Fenton-s-Reagent" target="_blank">treatment</a> of a persistant organic pollutant -- a relatively harmless one, 4-nitrophenol -- and the application of the rate measurements to a model of industrial wastewater treatment under a cost constraint.
									How large a continuous flow reactor and what quantities of the components of Fenton's reagent (a ferrous iron compound and hydrogen peroxide) should be used to decontaminate the waste stream to an acceptable concentration?
								</p>

								<h3 class="major">Additional</h3>
								<p>
									<a href="images/Acetone_Fermentation_History.pdf"><u>Here</u></a> is a presentation that I made on an interesting episode in scientific, military, and political history: the discovery of acetone-butanol-ethanol fermentation as a solution to the British munitions crisis in World War I.
								</p>
								
							</article>

						<!-- Contact -->
							<article id="contact">
								<h2 class="major">Contact</h2>
								<p>
									samuelmiller10 at gmail dot com<br />
									smiller at mbl dot edu
								</p>
								<!-- form method="post" action="#">
									<div class="fields">
										<div class="field half">
											<label for="name">Name</label>
											<input type="text" name="name" id="name" />
										</div>
										<div class="field half">
											<label for="email">Email</label>
											<input type="text" name="email" id="email" />
										</div>
										<div class="field">
											<label for="message">Message</label>
											<textarea name="message" id="message" rows="4"></textarea>
										</div>
									</div>
									<ul class="actions">
										<li><input type="submit" value="Send Message" class="primary" /></li>
										<li><input type="reset" value="Reset" /></li>
									</ul>
								</form -->
								<ul class="icons">
									<!-- li><a href="#" class="icon fa-twitter"><span class="label">Twitter</span></a></li -->
									<!-- li><a href="#" class="icon fa-facebook"><span class="label">Facebook</span></a></li -->
									<!-- li><a href="#" class="icon fa-instagram"><span class="label">Instagram</span></a></li -->
									<li><a href="https://github.com/semiller10" class="icon fa-github"><span class="label">GitHub</span></a></li>
									<!-- li><a href="https://www.linkedin.com/in/samuel-miller-866b27178/" class="icon fa-linkedin"><span class="label">LinkedIn</span></a></li -->
								</ul>
							</article>

						<!-- Elements -->
							<article id="elements">
								<h2 class="major">Elements</h2>

								<section>
									<h3 class="major">Text</h3>
									<p>This is <b>bold</b> and this is <strong>strong</strong>. This is <i>italic</i> and this is <em>emphasized</em>.
									This is <sup>superscript</sup> text and this is <sub>subscript</sub> text.
									This is <u>underlined</u> and this is code: <code>for (;;) { ... }</code>. Finally, <a href="#">this is a link</a>.</p>
									<hr />
									<h2>Heading Level 2</h2>
									<h3>Heading Level 3</h3>
									<h4>Heading Level 4</h4>
									<h5>Heading Level 5</h5>
									<h6>Heading Level 6</h6>
									<hr />
									<h4>Blockquote</h4>
									<blockquote>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan faucibus. Vestibulum ante ipsum primis in faucibus lorem ipsum dolor sit amet nullam adipiscing eu felis.</blockquote>
									<h4>Preformatted</h4>
									<pre><code>i = 0;

while (!deck.isInOrder()) {
    print 'Iteration ' + i;
    deck.shuffle();
    i++;
}

print 'It took ' + i + ' iterations to sort the deck.';</code></pre>
								</section>

								<section>
									<h3 class="major">Lists</h3>

									<h4>Unordered</h4>
									<ul>
										<li>Dolor pulvinar etiam.</li>
										<li>Sagittis adipiscing.</li>
										<li>Felis enim feugiat.</li>
									</ul>

									<h4>Alternate</h4>
									<ul class="alt">
										<li>Dolor pulvinar etiam.</li>
										<li>Sagittis adipiscing.</li>
										<li>Felis enim feugiat.</li>
									</ul>

									<h4>Ordered</h4>
									<ol>
										<li>Dolor pulvinar etiam.</li>
										<li>Etiam vel felis viverra.</li>
										<li>Felis enim feugiat.</li>
										<li>Dolor pulvinar etiam.</li>
										<li>Etiam vel felis lorem.</li>
										<li>Felis enim et feugiat.</li>
									</ol>
									<h4>Icons</h4>
									<ul class="icons">
										<li><a href="#" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
										<li><a href="#" class="icon fa-facebook"><span class="label">Facebook</span></a></li>
										<li><a href="#" class="icon fa-instagram"><span class="label">Instagram</span></a></li>
										<li><a href="#" class="icon fa-github"><span class="label">Github</span></a></li>
									</ul>

									<h4>Actions</h4>
									<ul class="actions">
										<li><a href="#" class="button primary">Default</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
									<ul class="actions stacked">
										<li><a href="#" class="button primary">Default</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
								</section>

								<section>
									<h3 class="major">Table</h3>
									<h4>Default</h4>
									<div class="table-wrapper">
										<table>
											<thead>
												<tr>
													<th>Name</th>
													<th>Description</th>
													<th>Price</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Item One</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Two</td>
													<td>Vis ac commodo adipiscing arcu aliquet.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Three</td>
													<td> Morbi faucibus arcu accumsan lorem.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Four</td>
													<td>Vitae integer tempus condimentum.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Five</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
											</tbody>
											<tfoot>
												<tr>
													<td colspan="2"></td>
													<td>100.00</td>
												</tr>
											</tfoot>
										</table>
									</div>

									<h4>Alternate</h4>
									<div class="table-wrapper">
										<table class="alt">
											<thead>
												<tr>
													<th>Name</th>
													<th>Description</th>
													<th>Price</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Item One</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Two</td>
													<td>Vis ac commodo adipiscing arcu aliquet.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Three</td>
													<td> Morbi faucibus arcu accumsan lorem.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Four</td>
													<td>Vitae integer tempus condimentum.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Five</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
											</tbody>
											<tfoot>
												<tr>
													<td colspan="2"></td>
													<td>100.00</td>
												</tr>
											</tfoot>
										</table>
									</div>
								</section>

								<section>
									<h3 class="major">Buttons</h3>
									<ul class="actions">
										<li><a href="#" class="button primary">Primary</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
									<ul class="actions">
										<li><a href="#" class="button">Default</a></li>
										<li><a href="#" class="button small">Small</a></li>
									</ul>
									<ul class="actions">
										<li><a href="#" class="button primary icon fa-download">Icon</a></li>
										<li><a href="#" class="button icon fa-download">Icon</a></li>
									</ul>
									<ul class="actions">
										<li><span class="button primary disabled">Disabled</span></li>
										<li><span class="button disabled">Disabled</span></li>
									</ul>
								</section>

								<section>
									<h3 class="major">Form</h3>
									<form method="post" action="#">
										<div class="fields">
											<div class="field half">
												<label for="demo-name">Name</label>
												<input type="text" name="demo-name" id="demo-name" value="" placeholder="Jane Doe" />
											</div>
											<div class="field half">
												<label for="demo-email">Email</label>
												<input type="email" name="demo-email" id="demo-email" value="" placeholder="jane@untitled.tld" />
											</div>
											<div class="field">
												<label for="demo-category">Category</label>
												<select name="demo-category" id="demo-category">
													<option value="">-</option>
													<option value="1">Manufacturing</option>
													<option value="1">Shipping</option>
													<option value="1">Administration</option>
													<option value="1">Human Resources</option>
												</select>
											</div>
											<div class="field half">
												<input type="radio" id="demo-priority-low" name="demo-priority" checked>
												<label for="demo-priority-low">Low</label>
											</div>
											<div class="field half">
												<input type="radio" id="demo-priority-high" name="demo-priority">
												<label for="demo-priority-high">High</label>
											</div>
											<div class="field half">
												<input type="checkbox" id="demo-copy" name="demo-copy">
												<label for="demo-copy">Email me a copy</label>
											</div>
											<div class="field half">
												<input type="checkbox" id="demo-human" name="demo-human" checked>
												<label for="demo-human">Not a robot</label>
											</div>
											<div class="field">
												<label for="demo-message">Message</label>
												<textarea name="demo-message" id="demo-message" placeholder="Enter your message" rows="6"></textarea>
											</div>
										</div>
										<ul class="actions">
											<li><input type="submit" value="Send Message" class="primary" /></li>
											<li><input type="reset" value="Reset" /></li>
										</ul>
									</form>
								</section>

							</article>

					</div>

				<!-- Footer -->
					<footer id="footer">
						<p class="copyright">&copy; Samuel E. Miller. Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
					</footer>

			</div>

		<!-- BG -->
			<div id="bg"></div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
